test/controller-test.js

```diff
@@ -0,0 +1,71 @@
+import {expect} from 'chai'
```

Comment:
@Sukriti-sood
Rename this folder to `tests`. 
Inside tests - Make 3 folders - `controllers` , `managers`, `services` 
Inside these folders - Make file - `opportunity-test.js`

Reason for doing this: 
By current configuration, you made a file `controller-test.js`, but in the future, there will be USER APIs(and maybe more) added as well, so the structure should be a generic one so that Inside `tests->controllers`, We can have two files -`opportunity-test.js` and `user-test.js`.

Reply:
@Manvityagi I have done that but having some confusion regarding path for test in package.json